### PR TITLE
Release Candidate v2023.2.2

### PR DESCRIPTION
**Pull request by ONECasey**
Typed cache & Node cleanup. (#4409) 
core, internal/configs: HIP28-v2 fee collection (#4410)
Minor: removed time.Sleep from tests. (#4412)
Mainnet Release Candidate 2023.1.2 (#4376) (#4414)
chore: merge main into dev (#4415)
internal/params: set validator code fix hard forks (#4413)
internal/params: schedule HIP28v2 + val code fix (#4416)
Dev fix conflicts. (#4417)
Remove dockerfile hardcoded version numbers (#4418)
Fix concurrent map access. (#4421)
add inspectdb to cli (#4426)
fix beacon-ness in legacy sync (#4428)
Use Header instead Block. (#4424)
Sync dev with main branch ( v2023.2.1 hotfix ) (#4435)
Feature: generating protoflles with docker. (#4427)
fix stream log issue, replace print log error with warning (#4433) 
hmy: Gas Price Oracle improvements (#4436)
change cache folder in sync and move it under data folder (#4438)
Consensus: reduce consensus initialization steps. (#4387)
Fixed race errors in downloader & consensus (#4442)
Fix panic. (#4440)
Snapshot integration and add cache to statedb (#4419)
Consensus: removed double initialization. (#4404)
Rebase dev to current main (#4443) 
Fix compile error. (#4444) 
Fixed bug with storing already cancelled context. (#4445)
build: update base distribution in Travis (#4451)
[github action] build and push harmony proto docker image (#4453) 
rpc: add net_listening function (#4446)

### CODE REVIEW

File: api/service/legacysync/downloader/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+# if command fails, try this
+# docker buildx create --use
+docker buildx build -t frozen621/harmony-proto:latest --platform linux/amd64,linux/arm64 -f Proto.Dockerfile --progress=plain .
```
Comment (MaxMustermann2) [inline api/service/legacysync/downloader/build.sh:4]: Can this be changed to `harmony-one` docker image?

**Review by adsorptionenthalpy — APPROVED** [review]
First round of tests ran successfully

File: api/service/legacysync/downloader/build.sh (hunk 1 above)
Comment (ONECasey) [reply api/service/legacysync/downloader/build.sh:4]: @sophoah , can we do this?

Comment (sophoah) [reply api/service/legacysync/downloader/build.sh:4]: @MaxMustermann2 i don't see why not if it is required, i'd just like to understand how where when is that build.sh is being used. Another PR explaining that would be great.

Comment (sophoah) [reply api/service/legacysync/downloader/build.sh:4]: @MaxMustermann2 i think what we wanted here. We definitely can.

Comment (sophoah) [reply api/service/legacysync/downloader/build.sh:4]: https://github.com/harmony-one/harmony/pull/4453

**Pull request merged**